.github/ISSUE_TEMPLATE/04-performance.yml

```diff
@@ -0,0 +1,51 @@
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to suggest a performance improvement to Mocha! Please note that Mocha is a widely used library with many millions of weekly downloads, many thousands of dependents, and many years of longstanding behavior. Every change to it is inherently risky and may break user edge cases that we have no way to learn of otherwise.
+
+        So, to accept a performance improvement, we'd need to see _measurable, non-negligible improvement_ in Mocha's real-world usage. That requires providing at least:
+
+        - Exhaustive comparison with other alternatives that proves this is the winning approach
+          - Disclosure of any personal affiliation with the proposed solution and alternatives
+        - For runtime performance: some kind of reproducible benchmark to demonstrate exactly what's different
+        - For dependency cleanups:
+          - The specific change to the dependency tree of a project with Mocha and no other dependencies
+          - The specific change to the dependency tree of a project with Mocha other common deps: maybe, chai, express, and sinon?
```

Comment:
Is `maybe` the name of a dependency? If not, it shouldn't have a comma after it. If so, let's remove the question mark at the end of the sentence

Reply:
oh, haha